pkg/lightning/mydump/csv_parser.go

```diff
@@ -297,10 +372,13 @@ outside:
 			parser.fieldIndexes = append(parser.fieldIndexes, len(parser.recordBuffer))
 			break outside
 		default:
-			if err = processDefault(firstByte); err != nil {
-				return nil, err
+			if prevToken == csvTokenDelimiter {
+				parser.logSyntaxError()
+				return nil, errors.AddStack(errUnexpectedQuoteField)
 			}
+			parser.appendCSVTokenToRecordBuffer(firstToken)
 		}
+		prevToken = firstToken
```

Comment:
This assign seems wrong if firstToken is delimiter

Reply:
if `firstToken` is csvTokenDelimiter at line 348 and no errors in lines 353-359, this assignment will set `prevToken` to csvTokenDelimiter and go on to next step. not sure what is wrong.

Reply:
I see. In this case, prevToken should be csvTokenCloseDelimiter so there is no different.